pom.xml

```diff
@@ -1277,9 +1277,9 @@
             </build>
         </profile>
 
-        <!-- Elasticsearch 5.2+ test environment (default) -->
+        <!-- Elasticsearch 5.6+ test environment (default) -->
```

Comment:
This profile really does work for 5.2 and above, not just 5.6 and above. It has to do with dialects and things that changed in Elasticsearch 5.2 (normalizers). Since we did not port dialects from Search 5 yet, it's a bit confusing, granted, but still: let's keep it that way until we merge everything into the main Search repository.

Reply:
Ah yes sorry, I made is so and then we discussed and changed that but forgot to update the comment.

Reply:
To be honest but I didn't get to the end of it, the idea was to support latest 5.6 and 6.0 and remove everything else. But we can have this discussion another time. Will revert.